homeassistant/components/matter/switch.py

```diff
@@ -263,6 +263,24 @@ def _update_from_device(self) -> None:
         ),
         vendor_id=(4874,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SWITCH,
+        entity_description=MatterNumericSwitchEntityDescription(
+            key="DoorLockEnablePrivacyModeButton",
+            entity_category=EntityCategory.CONFIG,
+            translation_key="privacy_mode_button",
+            device_to_ha={
+                0: False,
+                1: True,
+            }.get,
+            ha_to_device={
+                False: 0,
+                True: 1,
+            }.get,
```

Comment:
```suggestion
            device_to_ha=bool,
            ha_to_device=int,
```
Maybe this would work 🤔